src/EnergyPlus/CMakeLists.txt

```diff
@@ -612,3 +612,49 @@ if(UNIX AND NOT APPLE)
   COMMAND ${CMAKE_COMMAND} -E create_symlink energyplus EnergyPlus
   COMMAND ${CMAKE_COMMAND} -E rename EnergyPlus "${CMAKE_BINARY_DIR}/Products/EnergyPlus") 
 endif()
+
+# if we are building a package, we can do additional operations, such as run doc scripts
+if(BUILD_PACKAGE)
+  
+  # Some docs are generated on the fly here, create a dir for the 'built' files
+  set( DOCS_OUT "${CMAKE_BINARY_DIR}/autodocs" )
+  file( MAKE_DIRECTORY "${DOCS_OUT}" )
+  
+  # the output variables listing
+  add_custom_command( 
```

Comment:
I'll let you play with it if you want.  I do recall trying out the DEPENDS keyword and not getting it to respond correctly when I made changes in the depended-on files.  But that was a while back early in this branch's life; I might have better luck these days...